package/rules/file.js

```diff
@@ -1,8 +1,8 @@
 const { join } = require('path')
-const { source_path: sourcePath } = require('../config')
+const { source_path: sourcePath, static_assets_extensions: fileExtensions } = require('../config')
 
 module.exports = {
-  test: /\.(jpg|jpeg|png|gif|tiff|ico|svg|eot|otf|ttf|woff|woff2)$/i,
+  test: new RegExp(fileExtensions.join('|'), 'i'),
```

Comment:
Missing dot
```suggestion
  test: new RegExp(`\\.(${fileExtensions.join('|')})`, 'i'),
```

Reply:
@Morantron Not sure if we need dot there since extensions already include them? 

```js
/.jpg|.jpeg|.png|.gif|.tiff|.ico|.svg|.eot|.otf|.ttf|.woff|.woff2/i
```

Reply:
Changed it to this: `new RegExp(`(${fileExtensions.join('|')})`, 'i')`

Reply:
True! It's missing the trailing dollar, It could be an issue since without it it can match the extension in the middle of the filename

Reply:
Gotcha thanks @Morantron 🙏

Reply:
Fixed in #1832